frontend/my-app/src/Pages/CreateAccount.tsx

```diff
@@ -42,32 +52,105 @@ const CreateAccount: React.FC = () => {
-      <a href="/forgot-password" className="links">Forgot password?</a>
-      <a href="/" className="links">Login to existing account</a>
+    <div style={{ height: '90vh', margin: 0, padding: 0 }}>
+      <Stack
+        direction="column"
+        alignItems="center"
+        justifyContent="center"
+        spacing={6}
+        sx={{
+          height: 1
+        }}>
+        <Typography
+            component="h1"
+            variant="h1">
+            Equidistant
+        </Typography>
+        <Box
+          sx={{
+            marginTop: 8,
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+            mx: 'auto'
+          }}>
+          <Box
+            component="form"
+            onSubmit={handleCreate}
+            noValidate
+            sx={{
+              mt: 1
+            }}>
+              <Typography
+                align='center'
+                component="h1"
+                variant="h5"
+                sx={{ mb: 2, color: 'text.primary' }}>
+                Account Creation
+              </Typography>
+              <ThemeProvider theme={theme}>
+                <TextField
+                  required
+                  fullWidth
+                  id="email"
+                  label="Email Address"
+                  name="email"
+                  autoComplete="email"
+                  variant="standard"
+                  autoFocus
+                  onChange={(e) => { setEmail(e.target.value) }}
+                  color="info"
+                  sx={{
+                    input: { color: 'white' }
+                  }}
+                />
+                <TextField
+                  margin="normal"
+                  required
+                  fullWidth
+                  name="password"
+                  label="Password"
+                  type="password"
+                  id="password"
+                  autoComplete="current-password"
+                  variant="standard"
+                  onChange={(e) => { setPassword(e.target.value) }}
+                  color="info"
+                  sx={{ input: { color: 'white' } }}
+                />
+                <TextField
+                  margin="normal"
+                  required
+                  fullWidth
+                  name="address"
+                  label="Address"
+                  type="address"
+                  id="address"
+                  autoComplete="address"
+                  variant="standard"
+                  onChange={(e) => { setAddress(e.target.value) }}
+                  color="info"
+                  sx={{ input: { color: 'white' } }}
+                />
+              </ThemeProvider>
+            <Button
+              onClick={handleCreate}
+              fullWidth
+              variant="contained"
+              sx={{ mt: 3, mb: 2, bgcolor: 'secondary.main' }}
+            >
+              Create Account
```

Comment:
I think we'll need to add input verification on these inputs but that can be done later